### PR TITLE
Fixes  #206 issue regarding navbar

### LINKED ISSUE #206: Footer-navbar should be shown below while web search query.
At present, while performing a web search query the footer navbar should be at bottom of the page just like loklak-search.

screenshot from 2017-05-04 00-27-56

comparison with loklak search -

screenshot from 2017-05-04 00-28-18

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
screenshot from 2017-05-06 11-52-49

footer is fixed to the bottom as suggested

### CODE REVIEW

File: src/app/index/index.component.html [hunk 1]
```diff
@@ -17,9 +17,7 @@ <h2 class="yacy" id="greeting"></h2>
         <div id="search-bar">
             <app-search-bar></app-search-bar>
         </div>
```
Comment (Marauderer97) [inline src/app/index/index.component.html]: There seems to be some indentation error here, along with a few extra blank lines.

**Comment by mariobehling** [comment]
There should not be a line in the middle of the bar. Why is there a line?

**Comment by mariobehling** [comment]
@nikhilrayaprolu Please also mention the issue in the description in future => #206. Let's follow best practices together. Thanks.

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Also the bar seems to cause some problems in positioning on the index page, though it positions  properly in the results page. Please look into this.
image

**Pull request merged**